lib/node/http.js

```diff
@@ -295,6 +295,31 @@ class HTTP extends Server {
       res.json(200, filter.toJSON());
     });
 
+    // CfHeaders by startHeight, FilterType, and stopHash
+
+    this.get('/cfheaders/:type/:startHeight/:stopHash', async (req, res) => {
```

Comment:
Just curious, why did you add this REST API endpoint? We have the rpc commands already

Reply:
yeah you are right we don't need this endpoint.